runtime/default.policy

```diff
@@ -31,6 +54,43 @@ macros:
 # Rules determine what the Datadog security-agent will monitor.#
 ################################################################
 rules:
+## Process Activity Rules ##
+  - id: potential_web_shell
+    description: Potential webshell execution
+    expression: >-
+      (exec.filename in SHELLS || exec.filename in ["/usr/bin/curl", "/usr/bin/wget"] || exec.filename in SHELL_UTILS) && process.ancestors.name in WEBAPP_PROCESSES || process.ancestors.name =~ "php*"
+  
+  - id: suspicious_database_process
+    description: Shell or shell utilites spawned by a database process
+    expression: >-
+      (exec.filename in SHELLS || exec.filename in ["/usr/bin/curl", "/usr/bin/wget"] || exec.filename in SHELL_UTILS) && process.ancestors.name in DATABASE_PROCESSES || process.ancestors.name =~ "postgres*"
```

Comment:
same here

Reply:
you can use NET_UTILS here, or move curl and wget to HTTP_UTILS